numpy/lib/tests/test_function_base.py

```diff
@@ -1708,6 +1708,12 @@ def test_monotonic(self):
         bins = [1, 1, 0]
         assert_array_equal(digitize(x, bins, False), [3, 2, 0, 0])
         assert_array_equal(digitize(x, bins, True), [3, 3, 2, 0])
+        bins = [-1, 0, 1, 2]
+        assert_array_equal(digitize(x, bins, False, True), [1, 2, 3, 3])
```

Comment:
At least one of these should use kwargs to make the use clear
```suggestion
        assert_array_equal(digitize(x, bins, right=False, edge=True), [1, 2, 3, 3])
```

Reply:
The way it is now is stylistically consistent, I would put that in another PR that addresses style if you want it to look like that.